lib/client/client.go

```diff
@@ -419,28 +419,42 @@ func (c *NodeClient) RunInteractiveShell(ctx context.Context, mode types.Session
 
 // lineLabeledWriter is an io.Writer that prepends a label to each line it writes.
 type lineLabeledWriter struct {
-	linePrefix        []byte
-	w                 io.Writer
-	shouldWritePrefix bool
+	linePrefix    []byte
+	w             io.Writer
+	maxLineLength int
+
+	shouldWritePrefix   bool
+	currentLineProgress int
 }
 
-func newLineLabeledWriter(w io.Writer, label string) io.Writer {
+func newLineLabeledWriter(w io.Writer, label string, maxLineLength int) (io.Writer, error) {
+	prefix := []byte(fmt.Sprintf("[%v] ", label))
```

Comment:
nit: `fmt.Sprintf` is necessary here IMO as `label` is guaranteed to be a string and the format string is so simple

```suggestion
	prefix := "[" + label + "] "
```